### ISSUE TITLE
Style SVG children

### ISSUE THREAD

**Issue by bytrangle** [natemoo-re/astro-icon#79]
As I understand, you can only style the top-level `<svg>` element, the one with data attribute `[astro-icon]` like below:

```
<style>
    [astro-icon] {
        color: blue;
        /* OR */
        fill: blue;
    }
</style>
```

Targeting child of the SVG, like `path` element won't work:

```
<style lang="css">
  /* Nothing happens here*/
    [astro-icon] path {
        fill: blue;
    }
</style>
```

I would like to know if this is the intended behavior of Astro Icon. Is there any plan to allow for styling SVG children?

**Comment by stramel** [comment]
Thanks for bringing this up. Can you provide some more details or perhaps an example showing the issue? 

We have had a couple discussions about styling children before starting in #49. Let me know if you have a different issue.

**Comment by stramel** [comment]
should be the same way as in #49 

```css
svg path {
  fill: #aaa;
}
```

https://stackblitz.com/edit/github-ex6klk-31wfte?file=src/pages/index.astro

Closing as duplicate of #49
